common/static/common/js/components/views/paging_footer.js

```diff
@@ -17,23 +17,21 @@
                     this.collection.bind('add', _.bind(this.render, this));
                     this.collection.bind('remove', _.bind(this.render, this));
                     this.collection.bind('reset', _.bind(this.render, this));
-                    this.render();
```

Comment:
Yeah I agree @andy-armstrong, so there was a little inconsistency in `PagingHeader` and `PagingFooter` classes. One was rendering it in `initialize` and other wasn't, so one class was actually rendering itself two times.
@muzaffaryousaf ^^